src/components/MainSection.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Wrapper from "./Wrapper";
+import Arrow from "/src/img/arrow.png";
+import CircleButton from "/src/img/circle-button.svg";
+
+function MainSection() {
+  return (
+    <section className="bg-[url('/src/img/main-section-bg.jpg')] bg-cover bg-center w-full h-screen border-y border-secondary-color">
+      <Wrapper>
+        <div className="relative h-screen flex flex-col md:flex-row justify-center items-center">
+          <div className="md:grow text-sm md:text-base flex flex-col items-center space-y-1 text-center text-primary-dark">
+            <p>Lorem ipsum</p>
+            <h1 className="font-caption text-6xl md:text-8xl lg:text-[110px] text-brand-light-green">
+              Color Compass
+            </h1>
+            <p className="w-2/3 md:w-[460px]">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus
+              mattis tortor ut tortor cursus, ac maximus tortor.
+            </p>
+          </div>
+          <a
+            href="#"
+            className="md:grow-0 hover:animate-arrowRotate origin-left animate-pulse"
```

Comment:
I love the animations and the hover effect! It's an easter egg I didn't even notice at first but great touch =)

Reply:
Love the animation!